akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -507,6 +542,19 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
       Source.maybe[ByteString].via(killSwitch.flow))
   }
 
+  val inboundLargeFlow: Flow[ByteString, ByteString, NotUsed] =
```

Comment:
the materialized streams will be different, but this is for defining the `Flow` (blueprint)